### ISSUE TITLE
Input image for the training process

### ISSUE THREAD

**Issue by cciprianmihai** [vlbthambawita/singan-seg#2]
How did you create the `polyp_with_4_channels.png` image?

Can I use as `input image` an example from another dataset?

Thank you in advance!

**Comment by 1qaz2wsx3edc4rfv5tgb6yhn** [comment]
Yes, you can train on the image of your choice. I trained successfully using the supplied command line (I think you can drop the gpu arg if you only have cpu)

main_train.py --input_name <your_file_name_from_Input\Images>.png  --nc_z 4 --nc_im 4 --gpu_id 0

Generating random images from a trained model has been more difficult, digging around I found out that singan-seg appears based upon this work: https://github.com/tamarott/SinGAN/tree/master/SinGAN, I'm hoping to find more info there as my generated images look nothing like the trained model outputs during training.
